### PR TITLE
Updates staff list in the changelog

### PR DESCRIPTION
**Pull request by praisenarsie**

### CODE REVIEW

**Review by Carbonhell — DISMISSED** [review]

File: html/changelog.html [hunk 1]
```diff
@@ -35,14 +35,14 @@
...
+			<font size='2'><b>Host:</b> ErikHanson & AdmiralHippie<br></font>
+			<font size='2'><b>Head Admins:</b> DudeInYourBackyard, Carbonhell, tfw</font><br>
+			<font size='2'><b>Head Coders:</b> Pyko1 & ErikHanson <br></font>
+			<font size='2'><b>Game Masters:</b> BardOfRage, TeamStab, 2dollarbill, Looping, fluxcapacitor1337, tortellinitony, Maroonenahmoron<br></font>
+			<font size='2'><b>Game Admins:</b> Bartels, grxx, AbusiveToaster, Slehmerbo, Rikaru19xjenkins, Hackole, Lordkruor, Gandalf2k15, Brawzki, Oprayx73, swagile, Spankmaster, Woross<br></font>
+			<font size='2'><b>Code Maintainers:</b> TDCoolGuy, Pyko1, GuyonBroadway, JohnGinnane, ErikHanson, FrozenGuy5, tortellinitony, Banthebantz <br></font>
 			<font size='2'><b>Thanks to:</b> /tg/station, Baystation 12, /vg/station, NTstation, CDK Station devs, FacepunchStation, GoonStation devs, the original SpaceStation developers and Invisty for the title image.<br> Also a thanks to anybody who has contributed who is not listed here :( Ask to be added here on irc.</font>
+			<font size='2'><b>Rest In Peace: ArcLumin.
```
Comment (Manly-Dwarf) [inline html/changelog.html]: add a closing [/b]

Comment (JohnGinnane) [reply html/changelog.html]: impressive

Comment (JohnGinnane) [reply html/changelog.html]: `<\b>`

**Comment by JohnGinnane** [comment]
@Pyko1 @praisenarsie should it be byond usernames or github usernames?

**Comment by Pyko1** [comment]
byond really, the 1 is there since Pyko was already taken, most likely by myself.

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**